src/arithmetic/arithmetic_expression.c

```diff
@@ -210,7 +210,7 @@ static bool _AR_EXP_ValidateInvocation(AR_FuncDesc *fdesc, SIValue *argv, uint a
 	SIType expected_type = T_NULL;
 
 	// Make sure number of arguments is as expected.
-	if(fdesc->argc != VAR_ARG_LEN) {
+	if(fdesc->argc != VAR_ARG_LEN && fdesc->argc != AT_LEAST_ONE_VAR) {
```

Comment:
What about having a minimum and a maximum number of arguments for each function,
variable length function will set min to 0 and max to +inf,
in your case min would be 1 and max would be +inf

Reply:
done